src/services/item/plugins/importExport/index.ts

```diff
@@ -101,7 +102,7 @@ const plugin: FastifyPluginAsync = async (fastify) => {
   );
 
   // download item as zip
-  fastify.route<{ Params: { itemId: string } }>({
+  fastify.route<{ Params: { itemId: string }; Querystring: { type?: string } }>({
```

Comment:
If possible I would use a more descriptive argument name:
```suggestion
  fastify.route<{ Params: { itemId: string }; Querystring: { actionType?: string } }>({
```